rust-symcrypt/src/rsa/pss.rs

```diff
@@ -204,13 +207,13 @@ mod test {
             .unwrap();
 
         // tamper with signature
-        signature[0] = 0xFF;
+        signature[1] = 0xFF; // modifying second element, If we modify the first we will get InvalidArgument error instead.
```

Comment:
I'd need to look into the details, but I think it's likely that some non-trivial amount of the time (probably less than 1/256, but triggerable occasionally) the signature will have 0xFF as the second byte already, and this won't be a modification.

When trying to verify an invalid signature I would suggest two approaches:
1) Try to verify a signature signed by a distinct key of the same size
2) XOR a bit in the valid signature - i.e. this line becomes signature[1] ^= 0x1; - or something like that.

Reply:
Thanks for the advice, I will XOR instead